packages/reusable-blocks/package.json

```diff
@@ -37,6 +37,7 @@
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/icons": "file:../icons",
 		"@wordpress/notices": "file:../notices",
+		"@wordpress/private-apis": "^0.19.0",
```

Comment:
Any reason for this to not point to the local package like the other wp dependencies?

Reply:
PR to update path here: https://github.com/WordPress/gutenberg/pull/52712

Reply:
thanks for fixing @ramonjd